### PR TITLE
add: update octoxlabs

### PR DESCRIPTION
**Pull request by ogulcanhero**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Description
Octox Labs Cyber Security Asset Management platform XSoar Integration update.

## Minimum version of Cortex XSOAR
- [x] 6.0.0
- [x] 6.1.0
- [x] 6.2.0
- [x] 6.5.0
- [x] 6.12.0 

## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

**Comment by ogulcanhero** [comment]
Hi @kgal-pan, I'm running this command `demisto-sdk update-release-notes -i Packs/OctoxLabs` and I receive AttributeError for my added files (example: **Packs/OctoxLabs/Integrations/OctoxLabs/test_data/get_user.json**). Is any bug in demisto-sdk like this case?

**Comment by kgal-pan** [comment]
> Hi everyone, I'm running this command `demisto-sdk update-release-notes -i Packs/OctoxLabs` and I receive AttributeError for my added files (example: **Packs/OctoxLabs/Integrations/OctoxLabs/test_data/get_user.json**). Is any bug in demisto-sdk like this case?

Hey @ogulcanhero - I see that you already have a new release note created (`Packs/OctoxLabs/ReleaseNotes/2_0_0.md`). Please use that to fill out the changes you've introduced.

File: Packs/OctoxLabs/ReleaseNotes/2_0_0.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+
+#### Integrations
+
+##### OctoxLabs
```
Comment (Shellyber) [inline Packs/OctoxLabs/ReleaseNotes/2_0_0.md:4]: Please update the release notes with all the new commands

**Review by Shellyber — CHANGES_REQUESTED** [review]
Looking good.
Please update the release notes

**Comment by ogulcanhero** [comment]
Hi! I made the changes you requested. Is everything the way you want?

**Comment by ogulcanhero** [comment]
Okay, then if you tell us your available time, we can set a meeting with @ahmetkotan

File: Packs/OctoxLabs/Integrations/OctoxLabs/OctoxLabs.py [hunk 2]
```diff
@@ -256,6 +267,246 @@ def get_query_by_name(octox: OctoxLabs, args: Dict[str, Any]) -> CommandResults:
...
 
+def get_companies(octox: OctoxLabs, args: Dict[str, Any]) -> CommandResults:
+    count, companies = octox.get_companies(
+        page=args.get("page", 1),
+        search=args.get("search", ""),
+        size=args.get("size", 20),
+    )
+
+    return CommandResults(
+        outputs_prefix="OctoxLabs.Companies",
```
Comment (Shellyber) [inline Packs/OctoxLabs/Integrations/OctoxLabs/OctoxLabs.py:278]:
```suggestion
        outputs_prefix="OctoxLabs.Company",
```
You should do it for the rest of the commands as well (domains, users) etc.

File: Packs/OctoxLabs/Integrations/OctoxLabs/OctoxLabs.py [hunk 3]
```diff
@@ -256,6 +267,246 @@ def get_query_by_name(octox: OctoxLabs, args: Dict[str, Any]) -> CommandResults:
...
+
+def get_users(octox: OctoxLabs, args: Dict[str, Any]) -> CommandResults:
+    count, users = octox.get_users(
+        page=args.get("page", 1),
+        search=args.get("search", ""),
+        size=args.get("size", 20),
+    )
+
+    return CommandResults(
+        outputs_prefix="OctoxLabs.Users",
```
Comment (Shellyber) [inline Packs/OctoxLabs/Integrations/OctoxLabs/OctoxLabs.py:400]:
```suggestion
        outputs_prefix="OctoxLabs.User",
```

File: Packs/OctoxLabs/Integrations/OctoxLabs/OctoxLabs.py [hunk 4]
```diff
@@ -256,6 +267,246 @@ def get_query_by_name(octox: OctoxLabs, args: Dict[str, Any]) -> CommandResults:
...
+
+def get_domains(octox: OctoxLabs, args: Dict[str, Any]) -> CommandResults:
+    count, domains = octox.get_domains(
+        page=args.get("page", 1),
+        search=args.get("search", ""),
+        size=args.get("size", 20),
+    )
+
+    return CommandResults(
+        outputs_prefix="OctoxLabs.Domains",
```
Comment (Shellyber) [inline Packs/OctoxLabs/Integrations/OctoxLabs/OctoxLabs.py:339]:
```suggestion
        outputs_prefix="OctoxLabs.Domain",
```

**Comment by ogulcanhero** [comment]
Screenshot 2024-05-30 at 14 31 40

Hi @Shellyber! We run octoxlabs-get-companies, after that run octoxlabs-get-company-by-id. As you can see in the image above, the schemes look irregular in our opinion. It seems like it's better to keep it the way it was before in terms beacuse Companies schema is plural. Do you think we should change it anyway?

**Comment by Shellyber** [comment]
@ogulcanhero It's because when you run the `get-companies` command you return the result as is, instead of only what under the results. 
You can keep it as is if you don't want to change the entire command.

**Comment by ogulcanhero** [comment]
For now, we want to keep the command as it is. Older commands also had the same structure. We separate 'companies' and 'company' on purpose. Let’s proceed with that if it’s viable.

**Review by Shellyber — APPROVED** [review]

**Pull request merged**